packages/api/src/index.ts

```diff
@@ -36,9 +36,24 @@ export default {
     } catch (error: any) {
       if (error instanceof HTTPError) {
         return jsonResponse(JSON.stringify({ message: error.message }), error.status)
+      } else {
+
+        const status = error?.status || 500
+
+        let message: string;
+
+        if (typeof error === 'string') {
+          message = error
+        } else if (error instanceof Object) {
+          message = error.message || JSON.stringify(error)
+        } else {
+          message = `Uncaught error: ${error.toString()}`
+        }
+
+        const httpError = new HTTPError(message, status);
+        return jsonResponse(JSON.stringify({ message: httpError}), httpError.status)
```

Comment:
Is there any tests we could add to reach this code?

Reply:
I believe this error catch/response is tested with a few error tests here: https://github.com/web3-storage/w3name/blob/chore/no-error-swallowing/packages/api/test/name.spec.ts#L59